tests/system/test_misc.c

```diff
@@ -389,3 +389,98 @@ int cmd_exit_cb(int argc, char **argv)
    DEVSHELL_CMD_ASSERT(after_cb == before_cb + 1);
    return 0;
 }
+
+static bool
+comes_after(struct timeval t2,
+            struct timeval t1)
+{
+   // t1 must come before t2
+   return t2.tv_sec > t1.tv_sec || (t2.tv_sec == t1.tv_sec && t2.tv_usec > t1.tv_usec);
```

Comment:
Style: this line is 87 chars long, while the hard limit is 80 (see docs/contributing.md). Can you please split it?

Btw, you can run the following script from project's root to check for long lines easily: `./scripts/dev/count_lines_of_code -l`.  This way, you can find and fix any other "long" lines exceeding 80 cols.  At the moment, there are 0 lines exceeding 80 cols in the project, and I'm trying to keep that enforced unless there is a really good case for making an exception.